cmd/skywire-cli/commands/config/gen.go

```diff
@@ -67,7 +68,7 @@ func init() {
 	genConfigCmd.Flags().StringVarP(&hypervisorPKs, "hvpks", "j", "", "list of public keys to use as hypervisor")
 	genConfigCmd.Flags().StringVarP(&selectedOS, "os", "k", skyenv.OS, "(linux / macos / windows) paths")
 	genConfigCmd.Flags().BoolVarP(&stdout, "stdout", "n", false, "write config to stdout")
-	genConfigCmd.Flags().StringVarP(&output, "out", "o", "", "output config default:"+skyenv.ConfigName)
+	genConfigCmd.Flags().StringVarP(&output, "out", "o", skyenv.ConfigName, "output config default:"+skyenv.ConfigName)
```

Comment:
this change messes up the formatting of the help menu. The way it was before was intentional for that reason

![image](https://user-images.githubusercontent.com/36607567/162284438-0de0db5d-fe0d-4222-bdeb-51176ab15df1.png)

previously:

![image](https://user-images.githubusercontent.com/36607567/162284941-924ff40a-dcd4-4ecb-a996-b67e2d10761e.png)

Reply:
Fixed
![image](https://user-images.githubusercontent.com/38801159/162367666-24472ddd-4fa9-405f-aff3-1c6cd1ce96f7.png)